project/plugins.sbt

```diff
@@ -1,3 +1,3 @@
-addSbtPlugin("ch.epfl.scala" % "sbt-scalajs-bundler" % "0.18.0") // only needed for your JS dependencies, not for Laminar
+addSbtPlugin("ch.epfl.scala" % "sbt-scalajs-bundler" % "0.20.0") // only needed for your JS dependencies, not for Laminar
 
-addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.0.1")
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.1.0")
```

Comment:
scalajs-bundler 0.20.0 requires scalajs 1.3.0, so this 1.1.0 will be overwritten anyway. you should bump this to 1.3.1

Reply:
Also done on master.